runing/routers.py

```
@@ -0,0 +1,51 @@
+from fastapi import APIRouter, Depends, HTTPException
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)